api/sixtyfps-rs/migration.md

```diff
@@ -27,3 +27,12 @@ New code:
 ```rust,ignore
 let row_five = model.row_data(5).unwrap_or_default();
 ```
+
+`Model::model_tracker` has no default implementation anymore. This has no effect for custom dynamic models, as
+those have overridden the default implementation in any case. You will need to add this code into the implementation of the `Model` trait of your custom model:
+
+```rust,ignore
+fn model_tracker(&self) -> &dyn ModelTracker {
+    &()
+}
```

Comment:
Perhaps it would make sense to combine this change-set with the removal of `attach_peer`. Then this paragraph could be rewritten to guide the two use-cases:

"attach_peer() has been removed. Instead you must implement the `fn model_tracker(&self) -> &dyn ModelTracker` function. If you have a constant model, then you can just return `&()`, otherwise you can return a reference to the `ModelNotify` instance that you previously used in `attach_peer`:

Old code:

```rust
fn attach_peer(&self, peer: ...) {
    self.model_notify.attach_peer(peer);
}
```

New code:

```rust
fn model_tracker(&self) -> &dyn ModelTracker {
    &self.model_notify
}
```

or if your model is constant:

```rust
fn model_tracker(&self) -> &dyn ModelTracker {
    &()
}
```

(something along those lines :)